_tests/integration/reactnative_test.go

```diff
@@ -419,3 +442,184 @@ configs:
+            Click or drop your file on the upload file field\n1. Find the **CODE SIGNING
+            IDENTITY** section\n1. Click or drop your file on the upload file field\n1.
+            Click on **Workflows** tab\n1. Select deploy workflow\n1. Select **Xcode Archive
+            & Export for iOS** step\n1. Open **Force Build Settings** input group\n1. Specify
+            codesign settings\nSet **Force code signing with Development Team**, **Force
+            code signing with Code Signing Identity**  \nand **Force code signing with Provisioning
+            Profile** inputs regarding to the uploaded codesigning files\n1. Specify manual
+            codesign style\nIf the codesigning files, are generated manually on the Apple
+            Developer Portal,  \nyou need to explicitly specify to use manual coedsign settings
+            \ \n(as ejected rn projects have xcode managed codesigning turned on).  \nTo
+            do so, add 'CODE_SIGN_STYLE=\"Manual\"' to 'Additional options for xcodebuild
+            call' input\n\n## To run this workflow\n\nIf you want to run this workflow manually:\n\n1.
+            Open the app's build list page\n2. Click on **[Start/Schedule a Build]** button\n3.
+            Select **deploy** in **Workflow** dropdown input\n4. Click **[Start Build]**
+            button\n\nOr if you need this workflow to be started by a GIT event:\n\n1. Click
+            on **Triggers** tab\n2. Setup your desired event (push/tag/pull) and select
+            **deploy** workflow\n3. Click on **[Done]** and then **[Save]** buttons\n\nThe
+            next change in your repository that matches any of your trigger map event will
+            start **deploy** workflow.\n"
+          steps:
+          - activate-ssh-key@%s:
+              run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+          - git-clone@%s: {}
+          - script@%s:
+              title: Do anything with Script step
+          - yarn@%s:
+              inputs:
+              - command: install
+          - install-missing-android-tools@%s:
+              inputs:
+              - gradlew_path: $PROJECT_LOCATION/gradlew
+          - android-build@%s:
+              inputs:
+              - project_location: $PROJECT_LOCATION
+          - certificate-and-profile-installer@%s: {}
+          - xcode-archive@%s:
+              inputs:
+              - project_path: $BITRISE_PROJECT_PATH
+              - scheme: $BITRISE_SCHEME
+              - export_method: $BITRISE_EXPORT_METHOD
+              - configuration: Release
+          - deploy-to-bitrise-io@%s: {}
+        primary:
+          steps:
+          - activate-ssh-key@%s:
+              run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+          - git-clone@%s: {}
+          - script@%s:
+              title: Do anything with Script step
+          - yarn@%s:
+              inputs:
+              - command: install
+          - npm@%s:
```

Comment:
I think it should be `yarn test` instead.

Reply:
And if so do not forget to change the version to yarn here as well:
https://github.com/bitrise-io/bitrise-init/pull/167/files#diff-b2c12688a88be6d0a33b6badc95442dbR463